app/status/status.go

```diff
@@ -96,9 +92,13 @@ func readGroup(tomlFileName string) (*sda.Roster, error) {
 	return el, err
 }
 
+// printConn prints the status response that is returned from the server
 func printConn(e *status.Response) {
+	var a []string
 	for key, value := range e.Msg["Status"] {
-		log.Print(key + ": " + value)
+		a = append(a, (key + ": " + value + "\n"))
 	}
-	log.Print("\n")
+	sort.Strings(a)
+	strings.Join(a, "\n\n")
```

Comment:
This will give 3x `\n` per value, which is too much. One `\n` per value is nice.